doc/ref/chap-option.md

```diff
@@ -101,6 +101,7 @@ These options are from bash.
 ## Other Option
 
     noclobber   # Redirects don't overwrite files
+    errtrace    # Inherits ERR trap in shell functions
```

Comment:
same comment - describe how it changes the behavior for subprocesses?


Also mention that this is `-E`